cogs/hl.py

```diff
@@ -30,9 +30,20 @@ class Highlighter(AceMixin, commands.Cog):
 
 	@commands.command(aliases=['h1'])
 	@commands.bot_has_permissions(manage_messages=True, add_reactions=True)
-	async def hl(self, ctx, *, code):
+	async def hl(self, ctx, *, code: str = None):
 		'''Highlight some code.'''
-
+		if ctx.message.reference is not None:
+			# get the replied to message contents if there is one
+			ref_msg = ctx.message.reference.resolved
+			if ref_msg.author.bot:
+				raise commands.CommandError('Can not paste bot messages.')
+			code = ref_msg.content
```

Comment:
Assigning `code` here does nothing, as the following line overwrites any value of it:

https://github.com/Run1e/AceBot/blob/14288c53ee7432f417161813b4b075f07111bfad/cogs/hl.py#L50

Resulting in a code box only containing a language being sent.